common/SC_fftlib.cpp

```diff
@@ -300,7 +300,9 @@ scfft * scfft_create(size_t fullsize, size_t winsize, SCFFT_WindowFunction winty
 void scfft_ensurewindow(unsigned short log2_fullsize, unsigned short log2_winsize, short wintype)
 {
 	// Ensure we have enough space to do our calcs
+#if SC_FFT_FFTW
```

Comment:
@brianlheim can you explain why we can remove this?

Reply:
it's in the commit message - the variable is only used in code turned on by this preproc switch.